Dockerfile-base

```diff
@@ -24,7 +25,7 @@ RUN echo "selected_scheme scheme-basic" >> /install-tl-unx/texlive.profile; \
 RUN rm -r /install-tl-unx; \
 	rm install-tl-unx.tar.gz
 
-ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/texlive/2017/bin/x86_64-linux/
```

Comment:
I think we'll want to stay on texlive 2017 until we have a fully-tested upgrade in production.

(we still ship texlive 2017 in prod, and we intend to upgrade it in the next few months)